test-entrypoint.sh

```diff
@@ -1,13 +1,14 @@
-#!/bin/bash
+#!/bin/bash -e
```

Comment:
We may want `pipefail` set here, too, to make sure we exit non-zero if `go test` in the pipeline below fails.